ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -213,6 +213,29 @@
 {%         endif %}
 | {{ evpn_hostflap_detection_state }} | {{ evpn_hostflap_detection_window }} |  {{ evpn_hostflap_detection_threshold }} |
 {%     endif %}
+{%     set evpn_l2_gw_config = namespace(peer_groups = [], configured = false) %}
+{%     for peer_group in router_bgp.peer_groups | arista.avd.natural_sort %}
+{%         if router_bgp.address_family_evpn.peer_groups[peer_group].domain_remote is arista.avd.defined(true) %}
+{%             do evpn_l2_gw_config.peer_groups.append(peer_group) %}
+{%             set evpn_l2_gw_config.configured = true %}
+{%         endif %}
+{%     endfor %}
+{%     if evpn_l2_gw_config.configured is arista.avd.defined(true) %}
+
+#### EVPN DCI Gateway Summary
+
+| Settings | Value |
+| -------- | ----- |
+{%         if evpn_l2_gw_config.peer_groups | length > 0 %}
```

Comment:
This looks like you were doing double check, because ```evpn_l2_gw_config.configured``` should be true only if ```evpn_l2_gw_config.peer_groups | length > 0``` is true. As you checked the first already, I wouldn't recommend to go for second check again. Perform either one of these.

Reply:
Removed one of the checks.